packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/include/camera_avfoundation/QueueUtils.h

```diff
@@ -7,7 +7,7 @@
 NS_ASSUME_NONNULL_BEGIN
 
 /// Queue-specific context data to be associated with the capture session queue.
-extern const char* FLTCaptureSessionQueueSpecific;
+extern const char *FLTCaptureSessionQueueSpecific;
```

Comment:
This probably depends on the exact version of clang-format, due to changes in the logic for detecting whether a file is C++ or Obj-C.

Reply:
I think I'm using the latest clang-format version. What version is CI using? Or has this file just not been touched in a really long time?